src/sentry/api/endpoints/organization_pinned_searches.py

```diff
@@ -52,6 +53,18 @@ def put(self, request: Request, organization) -> Response:
             visibility=Visibility.OWNER_PINNED,
             values={"query": result["query"], "sort": result["sort"]},
         )
+
+        # This entire endpoint will be removed once custom views are GA'd
+        GroupSearchView.objects.create_or_update(
```

Comment:
Will we need similar logic for other saved issue searches?

Reply:
No, the intention is to only port over pinned saved searches to custom views - remaining saved searches will be persisted elsewhere